### ISSUE TITLE
`beijing-tumor` - add orientation-corrected-data (RPI) and matching resolution for SC-masks

### ISSUE THREAD

**Issue by louisfb01** [neuropoly/data-management#255]
`git-annex/datasets/beijing-tumor` has different resolutions between images and their SC-masks. It is also not in RPI.

I reoriented them to RPI and then saw the resolution differences, so I rescaled the masks to their associated image sizes with...
```
# Executed in both the root of the repo and in derivatives/labels:
for file in sub*/anat/*.nii.gz;do sct_image -i ${file} -setorient RPI -o ${file}; done

# Executed in the root of the repo:
shopt -s globstar; for file in **/*.nii.gz; do sct_register_mulitmodal -i derivatives/labels/${file%.nii.gz}_sc-mask.nii.gz -d ${file} -o derivatives/labels/${file%.nii.gz}_sc-mask.nii.gz -identity 1; done
```

I will do a PR soon with the changes.

**Comment by mguaypaq** [comment]
@louisfb01 you'll need write access to push your new branch and your git-annex metadata, so I gave it to you. If you want to check, you should see a line like this:
```
 R W	datasets/beijing-tumor
```
with a `W` when you do `ssh git@data.neuro.polymtl.ca`.

**Comment by valosekj** [comment]
Heads up! Different orientation and resolution between images and labels is also common for other datasets, for example, `canproco`. The reason is that images were preprocessed (reorientation and resampling), and then SC was segmented; see `canproco/README.md`:

> Note: T2w sagittal images were preprocessed before the spinal cord was segmented ('_T2w_seg-manual.nii.gz') and disc labels were identified ('_T2w_labels-manual.nii.gz').
Namely, reorientation to RPI and resampling to 0.8mm isotropic voxel were performed. This is why the original T2w images have different dimensions than segmentations and labels.
Preprocessing steps: https://github.com/ivadomed/canproco/blob/8e1b2c35f96eeeb3838b512dd93eba25e5a5e97a/scripts-t2w_csa/sct-preprocess_data.sh#L162-L170
For context, see https://github.com/neuropoly/data-management/issues/197

I think we should not modify the git-annex files. Instead, you can apply the same preprocessing steps on input images (reorientation and resampling) to match them with the labels.

**Comment by jcohenadad** [comment]
Indeed, thank you for catching this @valosekj

**Comment by sandrinebedard** [comment]
@louisfb01 the masks were cropped  too is that right?

**Comment by louisfb01** [comment]
Yes @sandrinebedard , the masks are cropped I believe (smaller than the images in all dimensions).
